builds.json

```diff
@@ -60,7 +60,7 @@
         {
           "package": "org.opencb.biodata.models",
           "python_package": "opencb",
-          "version": "1.2.1",
+          "version": "1.3.0-SNAPSHOT",
```

Comment:
Sorry I still don't quite get this, humour me... :stuck_out_tongue: 
Does this mean we'll no longer build 1.2.1 ?

Reply:
Correct. Let me use this PR to explain this in the README.md. One sec